activiti-cloud-services-runtime-bundle/activiti-cloud-services-rest-impl/src/main/java/org/activiti/cloud/services/rest/controllers/ProcessInstanceAdminControllerImpl.java

```diff
@@ -61,9 +68,32 @@ public ProcessInstanceAdminControllerImpl(ProcessInstanceResourceAssembler resou
                                                   pageConverter.toSpringPage(pageable, processInstancePage),
                                                   resourceAssembler);
     }
-
+    
+    private String getProcessDefinitionKey(StartProcessPayload startProcessPayload) {
```

Comment:
this seems to be repeated in the below class as well. Can be extracted in a common place?

Reply:
In admin area we use `processAdminRuntime` and in non admin  `processRuntime` 
`ProcessDefinition processDefinition = **processAdminRuntime**.processDefinition(startProcessPayload.getProcessDefinitionId());`
So we can make something common, but it will require more efforts and lines of code...
Maybe once we will make one general refactoring for admin and non-admin area...